src/app/modules/document-search/results/results.component.ts

```diff
@@ -144,14 +150,15 @@ export class ResultsComponent implements OnInit {
   translatedData: object;
 
   constructor(
-    @Inject(AUTH_SERVICE) private readonly authService: IAuthService,
+    private userService: UserService,
     private readonly router: Router,
     private readonly translateService: TranslateService,
     private readonly activeRoute: ActivatedRoute,
     private readonly attachmentUploadService: AttachmentUploadService
   ) {}
 
   ngOnInit() {
+    this.translateService.use(getBestMatchLanguage(this.userService));
```

Comment:
Not needed

Reply:
not using !! after adding addInitializerModule the translation is working proper.